src/connectedComponents/ViewerMain.js

```diff
@@ -139,9 +139,9 @@ class ViewerMain extends Component {
       <div className="ViewerMain">
         {this.state.displaySets.length && (
           <ConnectedLayoutManager
-          studies={this.props.studies}
-          viewportData={this.getViewportData()}
-          setViewportData={this.setViewportData}
+            studies={this.props.studies}
```

Comment:
Why are these whitespace changes in this PR?

Reply:
Sorry, the prettier plugin was enabled in my IDE and fixing style problems when saving the files. I already disabled but I forgot to reset the changes of this specific file. I will reset the code here.